src/tests/server.ts

```diff
@@ -962,83 +1073,103 @@ describe('Subscribe', () => {
 
 describe('Keep-Alive', () => {
   it('should dispatch pings after the timeout has passed', async () => {
-    await makeServer({
+    expect.assertions(1);
+    jest.useFakeTimers();
```

Comment:
This is also hard to test I tried `jest.advanceTimersByTime(49)` in order to prove that the ping is not sent earlier. However then I had to add multiple `await new Promise(setImmediate);` afterwards instead of a single one for the `callback` to be invoked.

The event loop timings with fake timers behave incomprehensible to me...

Reply:
Maybe it is easier to just skip the fake timers and just prove that the ping is invoked.